compiler/transforms/convert_stream_to_snax_stream.py

```diff
@@ -77,6 +76,31 @@ def match_and_rewrite(
         )
         schedule = scheduler(template, schedule)
 
+        schedule_op = stream.ScheduleOp(
+            op.inputs,
+            op.outputs,
+            ArrayAttr([AffineMapAttr(s.pattern) for s in schedule]),
+            rewriter.move_region_contents_to_new_regions(op.body),
+            schedule[0].bounds,
+            [[]],
+            op.accelerator,
+            op.result_types,
+        )
+
+        rewriter.replace_matched_op(schedule_op)
+
+
+@dataclass
+class LayoutResolution(RewritePattern):
+    @op_type_rewrite_pattern
+    def match_and_rewrite(self, op: stream.ScheduleOp, rewriter: PatternRewriter):
+        template = get_accelerator_info(op)
```

Comment:
If this were a method:
```suggestion
        template = op.get_accelerator_template()
```